service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/parameters/RequestStatementTemplateParameters.java

```diff
@@ -11,7 +17,19 @@
     private boolean isNested;
     private String requestStatementId;
     private String availabilityTime;
-    private String description;
     private String defaultReasonCode;
     private String code;
+    private String participant;
+    private String responsibleParty;
+    @Singular
+    private List<String> texts;
+
+    public Function<Object, String> compileText() {
```

Comment:
Why is this this reversed? I don't think this logic should be in our parameters object. All other parameters objects are logic-free.

Reply:
This seemed like the simplest way to build it - add to the list and then reverse it to simulate prepending. Fair shout re: logic. Is the only requirement that `ReferralRequest.description` be last, and all other content may be unordered before it?

Reply:
> the only requirement that ReferralRequest.description be last, and all other content may be unordered before it

that is correct